src/main/java/de/php_perfect/intellij/ddev/error_reporting/SentryErrorReporter.java

```diff
@@ -0,0 +1,112 @@
+package de.php_perfect.intellij.ddev.error_reporting;
+
+import com.intellij.diagnostic.AbstractMessage;
+import com.intellij.diagnostic.IdeaReportingEvent;
+import com.intellij.ide.DataManager;
+import com.intellij.ide.plugins.IdeaPluginDescriptor;
+import com.intellij.idea.IdeaLogger;
+import com.intellij.openapi.actionSystem.CommonDataKeys;
+import com.intellij.openapi.actionSystem.DataContext;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.diagnostic.ErrorReportSubmitter;
+import com.intellij.openapi.diagnostic.IdeaLoggingEvent;
+import com.intellij.openapi.diagnostic.SubmittedReportInfo;
+import com.intellij.openapi.progress.ProgressIndicator;
+import com.intellij.openapi.progress.Task;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.Messages;
+import com.intellij.util.Consumer;
+import io.sentry.Sentry;
+import io.sentry.SentryEvent;
+import io.sentry.protocol.SentryId;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
+
+public class SentryErrorReporter extends ErrorReportSubmitter {
+    @NotNull
+    @Override
+    public String getReportActionText() {
+        return "Report to Author";
```

Comment:
```suggestion
           return DdevIntegrationBundle.message("errorReporting.submit");
```
User translateable messages for all output presented to the user.
The text goes here: https://github.com/php-perfect/ddev-intellij-plugin/blob/509e09034b3538927616eca31acf0b3e7c4a1b3d/src/main/resources/messages/DdevIntegrationBundle.properties